res/values-ku/strings.xml

```diff
@@ -1023,7 +1023,7 @@
     <string name="POSITION_EJECT_BUTTON">POSITION EJECT BUTTON</string>
     <string name="POSITION_EJECT_TOGGLE_BUTTON">POSITION EJECT TOGGLE BUTTON</string>
     <string name="WIFI">WIFI</string>
-    <string name="JOIN_IP">JOIN IP</string>
+    <string name="JOIN_IP">HAT_IP"</string>
```

Comment:
You must remove the _ and the " now. On the translation, you made a good job.

Reply:
okey i will.